pkg/kubelet/stats/cri_stats_provider_windows.go

```diff
@@ -20,104 +20,102 @@ limitations under the License.
 package stats
 
 import (
-	"fmt"
 	"time"
 
+	"k8s.io/klog/v2"
+
 	"github.com/Microsoft/hcsshim"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/sets"
-	"k8s.io/klog/v2"
 	statsapi "k8s.io/kubelet/pkg/apis/stats/v1alpha1"
 )
 
+// hcsShimInterface creates an interface that allows for testing the logic without needing to create a container
+type hcsShimInterface interface {
```

Comment:
I'd prefer the windowsNetworkStatsInterface for this and any mentions of hcsshim also

Reply:
updated